Gems/ONNX/Code/Source/Clients/ONNXSystemComponent.h

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/Component/Component.h>
+#include <AzCore/Component/TickBus.h>
+#include <AzCore/IO/FileIO.h>
+#include <AzCore/std/smart_ptr/unique_ptr.h>
+#include <ImGuiBus.h>
+#include <LYImGuiUtils/HistogramGroup.h>
+#include <ONNX/ONNXBus.h>
+#include <imgui/imgui.h>
+
+namespace ONNX
+{
+    class ONNXSystemComponent
+        : public AZ::Component
+        , protected ONNXRequestBus::Handler
+        , public AZ::TickBus::Handler
```

Comment:
I recommend removing the `TickBus::Handler` stuff because this Gem does not use it.